### PR TITLE
specify file upload error status in same way as other appErrors

### PR DESCRIPTION
**Pull request by Baggerone**
Billy needs this change

### CODE REVIEW

File: application/models/file.go [hunk 1]
```diff
@@ -16,6 +16,12 @@ import (
 	"github.com/gofrs/uuid"
 )
 
+type FileUploadError struct {
```
Comment (Schparky) [inline application/models/file.go]: I like this implementation. I've never created a custom error type, but I'm curious why this doesn't have a `Error()` method.

File: application/actions/upload_test.go [hunk 2]
```diff
@@ -129,4 +126,46 @@ func (as *ActionSuite) Test_Upload() {
 	as.Regexp("^https?", u.URL)
 	as.Equal("image/gif", u.ContentType)
 	as.Equal(6, u.Size)
+
```
Comment (Schparky) [inline application/actions/upload_test.go]: Thanks for adding to this test.

File: application/models/file.go (hunk 1 above)
Comment (Baggerone) [reply application/models/file.go]: Funny enough, I wasn't thinking of it as implementing `error`.  I just gave it the attributes I wanted to be able to pull out of it.  I guess I could have given it and `Error()` method that just returns the `Message`. Do you think that's worth doing?

Comment (Schparky) [reply application/models/file.go]: Seems like it would make sense. Fewer lines affected maybe.

**Pull request merged**